extensions/data-plane/data-plane-http/README.md

```diff
@@ -25,9 +25,34 @@ Typically, this extension is used to fetch or post data from/to a REST endpoint.
 
 The setting parameters of this extension are listed below:
 
-| Parameter name                                      | Description                                                          | Mandatory | Default value |
-|:----------------------------------------------------|:---------------------------------------------------------------------|:----------|:--------------|
-| `edc.dataplane.http.sink.partition.size`  | Number of partitions for parallel message push in the `HttpDataSink` | false     | 5             |
+| Parameter name                           | Description                                                          | Mandatory | Default value |
+|:-----------------------------------------|:---------------------------------------------------------------------|:----------|:--------------|
+| `edc.dataplane.http.sink.partition.size` | Number of partitions for parallel message push in the `HttpDataSink` | false     | 5             |
+
+### Provided Services
+
+#### `HttpRequestParamsProvider`
+This service could be used to register HTTP params decorators (implementors of `HttpParamsDecorator`), that will be 
+invoked before the HTTP request creation, both on `source` and on `sink` sides.
```

Comment:
I was more thinking about this, but your example is also great :-) 

```suggestion
invoked before the HTTP request creation, both on `source` and on `sink` sides. This can be useful, to enhance the request
adding some headers with for example some data flow related info like "transferId" or "contractId".
```